### PR TITLE
Adds link in documentation to three.js manual re: creating BufferGeometries

### PR DESCRIPTION
**Pull request by DougReeder**
**Description:**

Custom three.js BufferGeometries are difficult to grok and the A-Frame documentation doesn't link to the very helpful manual article.

**Changes proposed:**

This adds links to the three.js manual page "Custom BufferGeometry" to the A-Frame docs.

### CODE REVIEW

File: docs/introduction/html-and-primitives.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ for [Don McCurdy's
 `aframe-physics-system`](https://github.com/n5ro/aframe-physics-system) and attach
 the physics components via HTML attributes:
 
-> :warning: **If you are using A-Frame 1.4.0 or later**: [`aframe-physics-system`](https://github.com/donmccurdy/aframe-physics-system) and you're having issues make sure you're no longer using the now deprecated THREE.Geometry. More info on [this GitHub issue](https://github.com/n5ro/aframe-physics-system/issues/187).
+> :warning: **If you are using A-Frame 1.4.0 or later**: [`aframe-physics-system`](https://github.com/donmccurdy/aframe-physics-system) and you're having issues make sure you're using THREE.BufferGeometry, not  the now-deprecated THREE.Geometry. (Recent versions of three.js rename BufferGeometry to just Geometry, confusingly.) More info on [this GitHub issue](https://github.com/n5ro/aframe-physics-system/issues/187).
```
Comment (dmarcos) [inline docs/introduction/html-and-primitives.md:130]: Can maybe link to the THREE release notes where THREE.Geometry was deprecated

File: docs/components/geometry.md [hunk 2]
```diff
@@ -319,7 +319,12 @@ the `material` component.
 ## Register a Custom Geometry
 
 We can register our own geometries using `AFRAME.registerGeometry` and creating
-an object that is an instance of [`THREE.Geometry`][three-geometry]. A-Frame
+an object that is an instance of [`THREE.BufferGeometry`][three-geometry].
+(Recent versions of three.js rename this to `Geometry`, which used to refer
+to unbuffered geometries.)
+See the [three.js manual](https://github.com/DougReeder/aframe/pull/new/buffer-geometry)
```
Comment (dmarcos) [inline docs/components/geometry.md:325]: correct link?

Comment (DougReeder) [reply docs/components/geometry.md:325]: good catch!

**Comment by dmarcos** [comment]
Thank you!

**Pull request merged**